ui/tests/integration/components/pki/pki-sign-intermediate-form-test.js

```diff
@@ -19,7 +19,7 @@ const selectors = {
   fieldByName: (name) => `[data-test-field="${name}"]`,
   saveButton: '[data-test-pki-sign-intermediate-save]',
   cancelButton: '[data-test-pki-sign-intermediate-cancel]',
-  fieldError: '[data-test-inline-alert]',
+  fieldError: '[data-test-validation-error="csr"]',
```

Comment:
oish, could we instead just use the GENERAL selector for this [here](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L121)